bl/evaluation/printer.go

```diff
@@ -170,8 +170,13 @@ func parseToPrinterWarnings(results *EvaluationResults, invalidYamlFiles []*extr
 
 			for _, ruleUniqueName := range rulesUniqueNames {
 				rule := rules[ruleUniqueName]
+				var fixLink string = ""
```

Comment:
You can do 2 different things:
* `var fixLink = ""`
* `var fixLink string` - GoLang default value for string is empty string

Reply:
sure will change that